cmd/minishift/cmd/delete.go

```diff
@@ -90,6 +112,7 @@ func removeInstanceConfigs() {
 }
 
 func init() {
-	viper.BindPFlags(deleteCmd.Flags())
+	deleteCmd.Flags().BoolVar(&forceMachineDeletion, "force", false, "Forces the deletion of the VM specific files in MINISHIFT_HOME.")
```

Comment:
You cannot get it all. As you say, it takes you a fair bit towards #405, but somewhere I had to stop. I needed this, since I grew tired of manually cleaning up while doing my testing.